### ISSUE TITLE
CLI with -w raises TypeError on file change

### ISSUE THREAD

**Issue by scanny** [microsoft/pyright#6742]
**Describe the bug**
After upgrading from `v1.1.325` to `v1.1.340`, running `pyright -w {target-path}` raises the following on the first file write (so second "watched" run, initial run works fine):
```
Watching for file changes...
TypeError: n.getFilePath is not a function
    at oA.<anonymous> (/Users/scanny/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/src/analyzer/service.ts:1255:40)
    at oA.emit (node:events:514:28)
    at oA.emitWithAll (/Users/scanny/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/node_modules/chokidar/index.js:541:32)
    at awfEmit (/Users/scanny/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/node_modules/chokidar/index.js:607:14)
    at /Users/scanny/.cache/pyright-python/1.1.340/node_modules/pyright/dist/pyright-internal/node_modules/chokidar/index.js:728:9
    at FSReqCallback.oncomplete (node:fs:201:5)
```

Downgrading to 1.1.325 eliminates this behavior.

**VS Code extension or command-line**
This is using the CLI from `zsh 5.8` on a Mac.

Let me know if there's anything else you need :)

**Comment by rchiodo** [comment]
Thanks for the issue. This was caused by the change to add URI support. Fix should be in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.341, which I just published.

**Comment by scanny** [comment]
Confirmed fixed on my Intel Mac running Big Sur and pyright v1.1.341.

Thanks @erictraut! :)
